DAL/Repositories/TeacherRepository.cs

```diff
@@ -167,8 +164,8 @@ select new
 
                                                         } );
 
-            string a = query.ToQueryString();
-            var aa = query.ToList();
+            string queryString = query.ToQueryString();
+            Console.WriteLine(queryString);
```

Comment:
For the future we need to log information with logger. Remember to add logger in such places for writing information.

Reply:
I got you. I will work in this direction.